README.md

```diff
@@ -135,30 +139,32 @@ This work was ported to Python from the original MATLAB developed by Dr. Veroniq
 Usage:
 
 ```console
-(rrap-dg) $ rrapdg dhw generate [input data directory] [output directory] [optional settings...]
+(rrap-dg) $ rrapdg dhw generate [cluster name] [input data directory] [output directory] [optional settings...]
 ```
 
 For example, with default values shown for optional settings:
 
 ```console
-(rrap-dg) $ rrapdg dhw generate C:/data_package_location C:/temp --n-sims 50 --rcps "2.6 4.5 6.0 8.5" --gen-year "2025 2100"
+(rrap-dg) $ rrapdg dhw generate Moore C:/data_package_location C:/temp --n-sims 50 --rcps "2.6 4.5 6.0 8.5" --gen-year "2025 2100"
 ```
 
 Note that the output directory is assumed to already exist.
 
 ## Initial Coral Cover
 
 Initial coral cover data is downscaled from ReefMod Engine (RME) data.
-The current process is compatible with RME v1.10.18
+The current process is compatible with ReefMod or RME v1.0.x datasets or the rrap-dg
+data package.
 
 ```console
-(rrap-dg) $ rrapdg coral-cover downscale-icc [reefmod engine datapackage path] [target geopackage] [output path]
+(rrap-dg) $ rrapdg coral-cover downscale-icc [rrap-dg datapackage path] [target geopackage] [output path]
 ```
 
 For example, to downscale RME data for the Moore cluster defined by a geopackage:
 
 ```console
-(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/RME_v1 ./Moore.gpkg ./coral_cover.nc
+(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/rrapdg ./Moore.gpkg ./coral_cover.nc
+(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/rme_dataset ./Moore.gpkg ./coral_cover.nc
```

Comment:
Does that also work with rme_dataset? Also, I think you need change `./Moore.gpkd` to `Moore`, no?

Reply:
Yes, it works with both the rrap-dg and RME darasets.

And, the idea now is to give it the geopackage file

Reply:
Oh, sorry, I missed on file 🤦🏻‍♂️